h/presenters/group_membership_json.py

```diff
@@ -0,0 +1,12 @@
+class GroupMembershipJSONPresenter:
+    def __init__(self, membership):
+        self.membership = membership
+
+    def asdict(self):
+        return {
+            "authority": self.membership.group.authority,
+            "userid": self.membership.user.userid,
+            "username": self.membership.user.username,
+            "display_name": self.membership.user.display_name,
+            "roles": self.membership.roles,
+        }
```

Comment:
This is compatible with the user objects currently returned by the [get-group-members API](https://h.readthedocs.io/en/latest/api-reference/v1/#tag/groups/paths/~1groups~1{id}~1members/get) just with `roles` added. (It was previously thought of as a list of users, but now we're thinking of it as a list of memberships.)